package.json

```diff
@@ -28,29 +28,29 @@
   "license": "private",
   "private": true,
   "dependencies": {
-    "@kubernetes/client-node": "^0.11.1",
-    "@types/async": "^3.0.8",
+    "@kubernetes/client-node": "^0.12.2",
+    "@types/async": "^3.2.3",
     "@types/child-process-promise": "^2.2.1",
     "@types/lru-cache": "^5.1.0",
     "@types/needle": "^2.0.4",
     "@types/node": "^10.17.17",
     "@types/sinon": "^7.5.2",
     "@types/tunnel": "0.0.1",
-    "async": "^2.6.2",
+    "async": "^3.2.0",
     "aws-sdk": "^2.633.0",
     "bunyan": "^1.8.13",
     "child-process-promise": "^2.2.1",
     "lru-cache": "^5.1.1",
     "needle": "^2.5.0",
     "sleep-promise": "^8.0.1",
-    "snyk-config": "3.0.0",
+    "snyk-config": "3.1.1",
     "snyk-docker-plugin": "2.8.0",
     "source-map-support": "^0.5.16",
     "tunnel": "0.0.6",
     "typescript": "^3.8.3",
     "ws": "^7.2.1",
     "yaml": "^1.7.2",
-    "snyk": "^1.320.3"
+    "snyk": "^1.413.5"
```

Comment:
assuming we have no patches - can't we remove this dependency and the .snyk file?